synapseAnnotations/data/analysis.json

```diff
@@ -395,143 +347,138 @@
         "source": "http://edamontology.org/operation_0292"
       },
       {
-        "value": "genotypeImputation", 
-        "description": "The statistical inference of unobserved genotypes", 
-        "source": ""
-      }, 
+        "value": "genotype imputation",
+        "description": "The statistical inference of unobserved genotypes.",
+        "source": "https://www.ncbi.nlm.nih.gov/pubmed/17572673"
+      },
       {
-        "value": "DNAmethylationImputation", 
-        "description": "", 
+        "value": "DNA methylation imputation",
+        "description": "The statistical inference of unobserved DNA methylation marks.",
         "source": "https://dx.doi.org/10.1534/genetics.115.185967"
       },
       {
-        "value": "polygenicRiskScore",
-        "description": "A sample level estimate of the genetic component of disease risk based on genome-wide association studies",
+        "value": "polygenic risk score",
+        "description": "A sample level estimate of the genetic component of disease risk based on genome-wide association studies.",
         "source": "http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003348"
       },
       {
-        "value": "Enrichment Analysis", 
-        "description": "Over-representation analysis", 
+        "value": "enrichment analysis",
```

Comment:
Preferred Name "Enrichment analysis"

Reply:
But that is not what you put in the file....